sechub-cli/src/mercedes-benz.com/sechub/util/filepathmatcher.go

```diff
@@ -3,44 +3,10 @@
 package util
 
 import (
-	"fmt"
 	"path/filepath"
-	"regexp"
 	"strings"
 )
 
-// FilePathMatch - This method provides ANT like selectors.
```

Comment:
Please keep the `FilePathMatch` function.
Using the bmatcuk/doublestar lib, it could look like this:
```Go
func FilePathMatch(path string, pattern string) (result bool) {
    match, _ := doublestar.PathMatch(pattern, path)
    return match
}
```